packages/netsuite-adapter/test/client/suiteapp_client/suiteapp_file_cabinet.test.ts

```diff
@@ -286,9 +286,14 @@ describe('suiteapp_file_cabinet', () => {
 
   describe('importFileCabinet', () => {
     const maxFileCabinetSizeInGB = 1
+    const extensionsToExclude = ['.*\\csv']
```

Comment:
Shouldn't it be the below?
Maybe we are missing another UT checking it?
```suggestion
    const extensionsToExclude = ['.*\\.csv']
```

Reply:
You're right. I'll fix it and add a test to `adapter.test.ts` where we get the list of excluded extensions.